cairo/Cargo.toml

```diff
@@ -28,11 +28,10 @@ freetype = ["ffi/freetype", "freetype-rs"]
 script = ["ffi/script"]
 xcb = ["ffi/xcb"]
 xlib = ["ffi/xlib"]
-dox = ["ffi/dox", "glib/dox", "freetype"]
 win32-surface = ["ffi/win32-surface"]
```

Comment:
Apparently, cairo did more than just enabling "dox" for its sys crate and dependencies, this seems to cause problems now.

Reply:
I think generally `dox` was also enabling all other relevant features, so you'd probably have to go with `--all-features` here now in addition.

Similarly, `dox` was enabling the `dox` feature of dependencies. The idea being that you want to have proper docs from e.g. `glib` too if you do `cargo doc` in `cairo`.